buildspec.yml

```diff
@@ -0,0 +1,39 @@
+version: 0.2
+
+phases:
+  install:
+    runtime-versions:
```

Comment:
Not necessary now, but you can also fix your node version to 16. It's the only version available in the 6.0 build containers, but you'll be alerted when/if that changes or if you switch pipeline versions.